odoo/orm/__init__.py

```diff
@@ -0,0 +1,37 @@
+# ruff: noqa: F401
```

Comment:
maybe rename to `_orm`, so it is clear that it is our internal stuff, that people should keep on importing from models/fields/api and not orm